blackmarket/UI/ViewModels/LogoutViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  LogoutViewModel.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 12/6/22.
+//
+
+import Foundation
+import Combine
+
+class LogoutViewModel: ObservableObject, Identifiable {
+    
+    private let isFetchingSubject = CurrentValueSubject<Bool, Never>(false)
+    private let errorSubject = CurrentValueSubject<String, Never>("")
+    private let authServices: AuthenticationServices
+    
+    var isValid: AnyPublisher<Bool, Never> {
+        CurrentValueSubject<Bool, Never>(true).eraseToAnyPublisher()
```

Comment:
is this ok? this value is not changing  ever

Reply:
I tried to reuse the component and this value is needed

Reply:
I think this is the wrong approach, adding extra code that serves no purpose in the view model other than a mandatory param in a UI component. Instead the StateButton could allow default parameters for isValid being `Just(true)` if nothing is provided.